src/App.tsx

```diff
@@ -15,36 +17,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+let goods = [...goodsFromServer];
+let isReverseCLicked = false;
+
 export const App: React.FC = () => {
+  const ALPHABETICALLY: string = 'alphabetically';
+  const BY_LENGTH: string = 'by length';
+  const RESET: string = 'reset';
+  const [sortGoods, setSortGoods] = useState('');
+  const [reverse, setReversed] = useState(false);
+
+  function fSort() {
```

Comment:
[CODE STYLE] The function `fSort` should be renamed to start with a verb, for example `sortGoods`.